src/wizard/index.js

```diff
@@ -1,10 +1,30 @@
 import Character from '../character';
 
-class Wizard {
+class Wizard extends Character{
+  #timesAttacked = 0;
+  #protection = 3;
```

Comment:
🏆 Nice making these private. I'm so used to only seeing private usage in Typescript that I did a double-take on these.